### PR TITLE
chore: add the 1.13.12/1.14.8/1.15.5 back.

### PR DESCRIPTION
**Pull request by andyliuliming**
**Reason for Change**:

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: pkg/api/common/versions.go [hunk 1]
```diff
@@ -146,7 +146,7 @@ var AllKubernetesSupportedVersions = map[string]bool{
 	"1.15.2":         false,
 	"1.15.3":         false,
 	"1.15.4":         true,
-	"1.15.5":         false, // disabled because of https://github.com/Azure/aks-engine/issues/2312
+	"1.15.5":         true, // disabled because of https://github.com/Azure/aks-engine/issues/2312, reenable because we patched it already.
```
Comment (andyzhangx) [inline pkg/api/common/versions.go]: actually it's a little tricky that the private patch is here: https://github.com/Azure/kubernetes/releases/tag/v1.15.5-hotfix.20191113
not sure whether aks-engine could adopt this.

Comment (xuto2) [reply pkg/api/common/versions.go]: this is for aks consumption only. we just need to make sure images in the below repo are updated with the patch.
	KubernetesImageBaseWithGlobalACR = "aksrepos.azurecr.io/mirror/"
	KubernetesImageBaseWithGlobalMCR = "mcr.microsoft.com/oss/kubernetes/"

Comment (xuto2) [reply pkg/api/common/versions.go]: Does the *_f0.0.2 patched images include the required change from Andy?

defaultValue: "v1.12.8_v0.0.5,v1.13.10_v0.0.5,v1.13.11_v0.0.5,v1.13.12_f0.0.2,v1.14.6_v0.0.5,v1.14.7_v0.0.5,v1.14.8_f0.0.2,v1.15.3_v0.0.5,v1.15.4_v0.0.5,v1.15.5_f0.0.2"

**Comment by xuto2** [comment]
I just cloned a new branch https://github.com/Azure/aks-engine/tree/aks-release-v0.45.0-1. Can you merge this PR to this branch and also vendor from there?

The convention is that the normal "aks-release-v0.45.0" branch should be based on aks-engine "release-v0.45.0". The change we're making here is monkey patches that should be identified separately.

File: pkg/api/common/versions.go (hunk 1 above)
Comment (andyliuliming) [reply pkg/api/common/versions.go]: @xuto2 yes, the f0.0.2 contains the fix from Andy.

@andyzhangx this is only for aks release.

**Pull request merged**